modules/specialized-guides/pages/salt/salt-custom-states.adoc

```diff
@@ -141,9 +141,11 @@ If you are bootstrapping Salt clients from the {productname} {webui}, you can us
 .Procedure: Trusting a GPG Key With a Custom Salt State
 . Locate the key that you need to trust.
   Ensure you have the correct key, and that you also have the fingerprint used to verify the key.
-  This information is available from the vendor or, in some cases, from a key server.
+This information is available from the vendor or, in some cases, from a key server.
 . Copy the key to a file location where the client can access it.
   We recommend saving it in the [path]``/srv/www/htdocs/pub/`` directory, where all {suse} public keys are also saved.
```

Comment:
customer gpg keys should be saved in `/srv/www/htdocs/pub/` but suse public key are now in `/usr/share/susemanager/gpg/`. Maybe we can change line to:

```
  Public key should be saved to [path]``/srv/www/htdocs/pub/`` directory, where all  user public files must be saved 
```
`/srv/www/htdocs/pub/` folder will be persisted in uyuni container, so this is the place where customer should save all the public file (reachable by UYUNI_FQDN/pub )

Reply:
yes, thanks for the proposal!